### PR TITLE
style: updating collection page for mobile

### PR DESCRIPTION
**Pull request by JackShort**
* updating mobile collection page styles to match figmas

### CODE REVIEW

File: src/nft/components/collection/CollectionNfts.tsx [hunk 1]
```diff
@@ -439,12 +466,13 @@ export const CollectionNfts = ({ contractAddress, collectionStats, rarityVerifie
                   isMobile={isMobile}
                   isFiltersExpanded={isFiltersExpanded}
                   onClick={() => setFiltersExpanded(!isFiltersExpanded)}
-                  collectionCount={collectionNfts?.[0]?.totalCount ?? 0}
                 />
               </TraceEvent>
-              <SortDropdown dropDownOptions={sortDropDownOptions} />
+              <SortDropodownContainer isFiltersExpanded={isFiltersExpanded}>
```
Comment (gbugyis) [inline src/nft/components/collection/CollectionNfts.tsx:471]: Tiny typo: SortDropodown -> SortDropdown

**Review by gbugyis — COMMENTED** [review]
Couple questions: 

1. Is the Sweep button supposed to collapse to icon-only like the Filter button? It isn't right now, but Figma looks like its supposed to. 

2. For collection stats I'm only seeing Total Volume - missing Global Floor and Floor 24h. Could be a data issue tho

(DMing screenshots)

File: src/nft/components/collection/CollectionStats.tsx [hunk 2]
```diff
@@ -295,34 +295,34 @@ const StatsRow = ({ stats, isMobile, ...props }: { stats: GenieCollection; isMob
...
-            <StatsItem label="24-Hour Floor" isMobile={isMobile ?? false}>
+            <StatsItem label="24-Hour Floor" shouldHide={false}>
               <PercentChange>
                 {floorChangeStr}% {arrow}
               </PercentChange>
             </StatsItem>
           ) : null}
           {totalSupplyStr ? (
-            <StatsItem label="Items" isMobile={isMobile ?? false}>
+            <StatsItem label="Items" shouldHide={isMobile ? true : false}>
```
Comment (aballerr) [inline src/nft/components/collection/CollectionStats.tsx:310]: can just be ```isMobile```

File: src/nft/components/collection/CollectionStats.tsx [hunk 3]
```diff
@@ -295,34 +295,34 @@ const StatsRow = ({ stats, isMobile, ...props }: { stats: GenieCollection; isMob
...
               {totalSupplyStr}
             </StatsItem>
           ) : null}
           {uniqueOwnersPercentage ? (
-            <StatsItem label="Unique owners" isMobile={isMobile ?? false}>
+            <StatsItem label="Unique owners" shouldHide={isMobile ? true : false}>
```
Comment (aballerr) [inline src/nft/components/collection/CollectionStats.tsx:315]: same here

File: src/nft/components/collection/CollectionStats.tsx [hunk 4]
```diff
@@ -295,34 +295,34 @@ const StatsRow = ({ stats, isMobile, ...props }: { stats: GenieCollection; isMob
...
           ) : null}
           {stats.stats?.total_volume ? (
-            <StatsItem label="Total Volume" isMobile={isMobile ?? false}>
+            <StatsItem label="Total Volume" shouldHide={false}>
               {totalVolumeStr} ETH
             </StatsItem>
           ) : null}
           {stats.stats?.total_listings && listedPercentageStr > 0 ? (
-            <StatsItem label="Listed" isMobile={isMobile ?? false}>
+            <StatsItem label="Listed" shouldHide={isMobile ? true : false}>
```
Comment (aballerr) [inline src/nft/components/collection/CollectionStats.tsx:325]: and here

File: src/nft/components/collection/FilterButton.tsx [hunk 5]
```diff
@@ -1,95 +1,46 @@
...
-            </Box>
-          )}
-          <Box className={buttonTextMedium} paddingLeft={!isFiltersExpanded ? '12' : '2'}>
-            {collectionCount > 0 ? putCommas(collectionCount) : 0} results
-          </Box>
-        </Box>
-      ) : null}
+      <FilterIcon
+        style={{
+          color: 'textPrimary',
```
Comment (aballerr) [inline src/nft/components/collection/FilterButton.tsx:40]: I think we can actually remove this inline style and make sure ```filterButton``` is textPrimary

**Review by gbugyis — APPROVED** [review]

**Pull request merged**